src/flamenco/runtime/tests/fd_exec_instr_test.c

```diff
@@ -1665,41 +1665,28 @@ fd_exec_vm_syscall_test_run( fd_exec_instr_test_runner_t * runner,
   uchar * rodata = input->vm_ctx.rodata ? input->vm_ctx.rodata->bytes : NULL;
   ulong rodata_sz = input->vm_ctx.rodata ? input->vm_ctx.rodata->size : 0UL;
 
-  /* Concatenate the input data regions into the flat input memory region */
-  ulong input_data_sz = 0;
-  for ( ulong i=0; i<input->vm_ctx.input_data_regions_count; i++ ) {
-    if( !input->vm_ctx.input_data_regions[i].content ) {
+  /* Load input data regions */
+  fd_vm_input_region_t * input_regions = fd_valloc_malloc( valloc, alignof(fd_vm_input_region_t), sizeof(fd_vm_input_region_t) * input->vm_ctx.input_data_regions_count );
+  ulong input_data_offset = 0UL;
+  for( ulong i=0; i<input->vm_ctx.input_data_regions_count; i++ ) {
+    fd_exec_test_input_data_region_t const * region = &input->vm_ctx.input_data_regions[i];
+    pb_bytes_array_t * array = region->content;
+    if( !array ) {
       continue;
     }
```

Comment:
@ibhatt-jumptrading are empty regions (ie., region_sz 0) possible? Should I not `continue`?

Reply:
It looks like they are possible (https://github.com/anza-xyz/agave/blob/3072c1a72b2edbfa470ca869f1ea891dfb6517f2/programs/bpf_loader/src/serialization.rs#L105)

Reply:
https://github.com/anza-xyz/agave/blob/3072c1a72b2edbfa470ca869f1ea891dfb6517f2/programs/bpf_loader/src/serialization.rs#L136 they are not possible